index.d.ts

```diff
@@ -7,5 +7,5 @@ export declare function testConnection(uri: string): Promise<string>
 export declare class PlainTextAuthProvider {
   id: number
   static new(): PlainTextAuthProvider
-  static test(): number
+  test(): number
```

Comment:
Why this change?

Reply:
Apparently rust part wasn't recompiled on previous PR to main, so that's the expected compilation output for this file.

Reply:
Is it a question whether we have such step or a request to add it?

Reply:
This is a request (I corrected my comment).

Reply:
Sure. It should be working now.